js/jquery/footer.js

```diff
@@ -0,0 +1,44 @@
+/* global document */
+
+import $ from 'jquery'
+import registerPlugin from './register-plugin'
+
+class Footer {
```

Comment:
Let's try to make this obsolete. We have such menu components in several places (ex. in the main menu, in the mobile burger menu) and could write a `menu` plugin that works in all different scenarios. I'll have a look if I can find some already existing code.